src/App.js

```diff
@@ -37,35 +49,110 @@ function AppNavbar() {
             />{" "}
           </Link>
         </Navbar.Brand>
-        <PWAInstallButton />
+        <Dropdown className="dropdown--translation">
+          <Dropdown.Toggle as={IconToggle} />
+          <Dropdown.Menu>
+            {Object.keys(AVAILABLE_LANGUAGES).map(key => {
+              const langValue = AVAILABLE_LANGUAGES[key];
+              return (
+                <Dropdown.Item
+                  key={langValue}
+                  active={currentLanguage === langValue}
+                  onClick={() => setLanguage(langValue)}
+                >
+                  {langValue}
+                </Dropdown.Item>
+              );
+            })}
+          </Dropdown.Menu>
+        </Dropdown>
+        {/* <PWAInstallButton /> */}
```

Comment:
since the top bar is a bit crowded, i'm wondering if we can move this language dropdown to the footer, and keep the install button for now. 

in places like brazil and poland, localization has a clear win - the entries we have from those places are in local languages, despite the app being in english. this also means we can rely upon their default locales to be reliable, and we can pick that up automatically inside `getDefaultLanguage`

in india, the default locale is not reliable, and we need a switcher. both pwa install and localization are features to increase engagement, but we don't know which one is more important. the pwa button has been successful so far (we know through ga events), and so i'm hesitant to giving this spot to something else.

my suggestion would be to keep this in the footer for now, track ga events to see how many people switch over, and then take a call on where we want to place it.

Reply:
👍

Reply:
fixed